YvesIraguha/personalFinanceMobile
# #171076307 View Expense details

#### What does this PR do?
- Allow a user to view expense details
#### Description of Task to be completed?
- Rebuild the home page
- Enable a user to view home page
#### How should this be manually tested?
- Clone the repo and checkout the branch `ft-view-expense-detials-#171076307`
- checkout the respective branch, and run `yarn && yarn start` to start the application command
- Follow the directions provided to open the application in the simulator or real device
-  Ensure that your back-end is working. 
#### What are the relevant pivotal tracker stories?
#171076307
